src/System.Management.Automation/engine/ExperimentalFeature/ExperimentalFeature.cs

```diff
@@ -120,6 +120,9 @@ static ExperimentalFeature()
                 new ExperimentalFeature(
```

Comment:
@SteveL-MSFT, your last commit had 3 failures in `PowerShell-CI-macos`
Native UNIX globbing tests.Globbing should not happen with quoted expressions
```powershell
Expected strings to be the same, but they were different.
Expected length: 54
Actual length:   57
Strings differ at index 53.
Expected: '...7/abc*'
But was:  '...7/abc.txt'
at <ScriptBlock>, /Users/runner/runners/2.165.2/work/1/s/test/powershell/Language/Scripting/NativeExecution/NativeUnixGlobbing.Tests.ps1: line 59
59: 		/bin/echo "$v" | Should -BeExactly $v
```
Native UNIX globbing tests.~ should not be replaced when quoted
```powershell
Expected strings to be the same, but they were different.
Expected length: 1
Actual length:   13
Strings differ at index 0.
Expected: '~'
But was:  '/Users/runner'
at <ScriptBlock>, /Users/runner/runners/2.165.2/work/1/s/test/powershell/Language/Scripting/NativeExecution/NativeUnixGlobbing.Tests.ps1: line 131
131: 		/bin/echo "~" | Should -BeExactly '~'
```
Native UNIX globbing tests.Should not expand quoted strings: "*"
```powershell
Expected strings to be the same, but they were different.
Expected length: 1
Actual length:   283
Strings differ at index 0.
Expected: '*'
But was:  'assets CHA...'
at <ScriptBlock>, /Users/runner/runners/2.165.2/work/1/s/test/powershell/Language/Scripting/NativeExecution/NativeUnixGlobbing.Tests.ps1: line 82
82:         Invoke-Expression "/bin/echo $arg" | Should -BeExactly '*'
```